### PR TITLE
Update to PureScript v0.15.0

### PR DESCRIPTION
**Pull request by sigma-andex**
**Description of the change**

Backlinking to purescript/purescript#4244

Migrates FFI to ES modules

---

**Checklist:**

- [x] Added the change to the changelog's "Unreleased" section with a reference to this PR (e.g. "- Made a change (#0000)")
- [x] Linked any existing issues or proposals that this pull request should close
- [ ] Updated or added relevant documentation
- [ ] Added a test for the contribution (if applicable)

### CODE REVIEW

**Comment by sigma-andex** [comment]
🏓 @JordanMartinez @thomashoneyman

**Review by thomashoneyman — APPROVED** [review]

**Review by JordanMartinez — APPROVED** [review]

**Comment by JordanMartinez** [comment]
@sigma-andex Looks like the `.eslintrc.json` file didn't get updated.

**Comment by sigma-andex** [comment]
Sorry I oversaw that one. I updated the eslintrc but I forgot how to update your script. All four matches failed

File: .eslintrc.json [hunk 1]
```diff
@@ -1,12 +1,9 @@
 {
   "parserOptions": {
-    "ecmaVersion": 5
+    "ecmaVersion": 6,
+    "sourceType": "module"
   },
   "extends": "eslint:recommended",
-  "env": {
-    "commonjs": true,
-    "node": true
```
Comment (JordanMartinez) [inline .eslintrc.json:8]: The `node` line should still be kept.

**Comment by JordanMartinez** [comment]
> Sorry I oversaw that one. I updated the eslintrc but I forgot how to update your script. All four matches failed

No worries! I didn't exactly document that part initially and then changed how it worked a few days ago.

File: .eslintrc.json (hunk 1 above)
Comment (sigma-andex) [reply .eslintrc.json:8]: Fixed

**Review by JordanMartinez — APPROVED** [review]

**Pull request merged**